src/server/models/statistics/clicks.ts

```diff
@@ -15,17 +15,36 @@ type UrlClicksTypeStatic = typeof Sequelize.Model & {
   new (values?: object, options?: Sequelize.BuildOptions): UrlClicksType
 }
 
-export const UrlClicks = <UrlClicksTypeStatic>sequelize.define('url_clicks', {
-  shortUrl: {
-    type: Sequelize.STRING,
-    primaryKey: true,
-    validate: {
-      is: SHORT_URL_REGEX,
+export const UrlClicks = <UrlClicksTypeStatic>sequelize.define(
+  'url_clicks',
+  {
+    shortUrl: {
+      type: Sequelize.STRING,
+      primaryKey: true,
+      validate: {
+        is: SHORT_URL_REGEX,
+      },
+    },
+    clicks: {
+      type: Sequelize.INTEGER,
+      defaultValue: 0,
+      allowNull: false,
     },
   },
-  clicks: {
-    type: Sequelize.INTEGER,
-    defaultValue: 0,
-    allowNull: false,
+  {
+    defaultScope: {
+      useMaster: true,
+    },
+    scopes: {
+      /**
+       * Use the master database for read queries. To be enabled
+       * when realtime data is needed.
+       */
+      useMasterDb() {
+        return {
+          useMaster: true,
+        }
+      },
+    },
```

Comment:
no need to define an additional `useMasterDb` scope, just use the `defaultScope` instead